Purchases.xcodeproj/project.pbxproj

```diff
@@ -2163,6 +2313,14 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
+		A592DAD026D83EF70065C78B /* XCRemoteSwiftPackageReference "purchases-ios" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/RevenueCat/purchases-ios.git";
```

Comment:
This setup might lead us to issues: since this is always pointing to `main`, we'll only realize when we're breaking the API _after_ the changes have been merged to `main`. If we have big feature branches, we might not realize something went wrong until it's too late. 

The ideal setup is to have the API tester build off of the `Purchases` instance in the same project. 

To achieve this, we need to: 
1. remove the SPM dependency 
<img width="595" alt="Screen Shot 2021-09-03 at 11 26 46" src="https://user-images.githubusercontent.com/3922667/132022267-5bf3ed38-9cd0-4bc7-a4c1-2b3e31fe043c.png">

2. Add an explicit dependency from SwiftAPITester to the `Purchases` target in Xcode, so that Xcode knows that it needs to build `Purchases` first, and re-build if there have been any changes made to `Purchases`. 
3. Link the framework, so it knows where to find `Purchases` when it gets imported. 
<img width="542" alt="Screen Shot 2021-09-03 at 11 27 04" src="https://user-images.githubusercontent.com/3922667/132022494-8da7a1c5-7ebb-4f7c-8194-bf2d0884b303.png">

This will ensure that we know as soon as the API change.

Reply:
The idea here was that I could run the migration script on this tester to ensure all breaking changes were caught and to more closely mimic a user's setup. I missed that this got switched to main, was originally pointing to latest released version..and actually I think it'd be nice to always point to latest released version, so we can catch breaking changes that perhaps aren't called from our sample apps. Thoughts on that? Perhaps having it in a separate project makes more sense?

Reply:
ohh I see, that makes sense. Sounds good.